src/assets/src/components/edit.tsx

```diff
@@ -241,6 +241,7 @@ const MeetingInfoDialog = (props: MeetingInfoProps) => {
                 Attendees: {props.meeting.attendees.map(a => <UserDisplay user={a}/>)}
             </p>
         );
+    const agendaInfo = <p>Agenda: {props.meeting?.agenda}</p>
```

Comment:
I think this should be moved into `generalInfo`, since it applies to any Meeting regardless of its backend. Also, in the `generalInfo` block above, the type system will have already verified that `props.meeting` must exist, so there will be no need for a safe navigation operator (`meeting?.`).